incubator/java-microprofile/image/Dockerfile-stack

```diff
@@ -21,21 +21,20 @@ RUN mvn -Dmaven.repo.local=/mvn/repository install dependency:go-offline -DskipT
 
 WORKDIR /project/user-app
 
-ENV APPSODY_MOUNTS="~/.m2/repository:/mvn/repository;src:/project/user-app/src;pom.xml:/project/user-app/pom.xml"
+ENV APPSODY_MOUNTS="~/.m2/repository:/mvn/repository;src:/project/user-app/src;pom.xml:/project/user-app/pom.xml;target:/project/user-app/target"
 ENV APPSODY_DEPS=
 
 ENV APPSODY_WATCH_DIR=/project/user-app
 ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/target
 ENV APPSODY_WATCH_REGEX="^.*(.xml|.java|.properties)$"
+ENV APPSODY_USER_RUN_AS_LOCAL=true
 
 ENV APPSODY_INSTALL="../validate.sh && mvn -B -Dmaven.repo.local=/mvn/repository install -DskipTests && /project/install-dev-deps.sh"
 
-ENV APPSODY_RUN="JVM_ARGS='-javaagent:resources/javametrics-agent.jar' mvn -B -Dmaven.repo.local=/mvn/repository liberty:run"
-ENV APPSODY_RUN_ON_CHANGE="JVM_ARGS='-javaagent:resources/javametrics-agent.jar' mvn -Dmaven.repo.local=/mvn/repository package -DskipTests"
+ENV APPSODY_RUN="JVM_ARGS='-javaagent:resources/javametrics-agent.jar' mvn -B -Dmaven.repo.local=/mvn/repository liberty:dev -DskipTests"
 ENV APPSODY_RUN_KILL=false
 
-ENV APPSODY_DEBUG="JVM_ARGS='-javaagent:resources/javametrics-agent.jar' mvn -B -Dmaven.repo.local=/mvn/repository liberty:debug"
-ENV APPSODY_DEBUG_ON_CHANGE="JVM_ARGS='-javaagent:resources/javametrics-agent.jar' mvn -Dmaven.repo.local=/mvn/repository package -DskipTests"
+ENV APPSODY_DEBUG="JVM_ARGS='-javaagent:resources/javametrics-agent.jar' mvn -B -Dmaven.repo.local=/mvn/repository liberty:dev -DskipTests"
```

Comment:
Once https://github.com/appsody/appsody/issues/670 is fixed, we should remove `-DskipTests` from `APPSODY_RUN` and `APPSODY_DEBUG` so that when using appsody with `--interactive` mode, dev mode will let users press Enter to run tests.  

We should wait for that issue to be fixed first before making this change though.  If needed, this can be a separate PR for this stack.

Reply:
https://github.com/appsody/appsody/issues/670 is fixed in Appsody 0.5.1.  Please proceed to remove `-DskipTests` as mentioned above.  Then if using non-interactive mode, tests will run upon code changes.  If using interactive mode, tests will run on demand when pressing Enter.